.gitignore

```diff
@@ -5,3 +5,6 @@ pom.xml.versionsBackup
 pom.xml.next
 release.properties
 dependency-reduced-pom.xml
+.idea
```

Comment:
Those things:
 1. are unrelated to the PR
 2. should go in a global gitignore on your machine, or possibly in your `.git/info/exclude`: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore, https://git-scm.com/docs/gitignore